test/jasmine/tests/plot_interact_test.js

```diff
@@ -142,6 +150,119 @@ describe('Test plot structure', function() {
             });
         });
 
+        describe('scatter drag', function() {
+
+            var mock = require('@mocks/scatter_simple.json'),
```

Comment:
@etpinard Thanks! Should I delete it, use a preexisting one and adjust the range numbers being tested? Or more as a future reference? In this specific case I just wanted to use something that's pretty much identical across renderers (SVG vs WebGL) except of course `type: 'scattergl'` vs `'scatter'`.